requirements/dev.in

```diff
@@ -5,21 +5,21 @@ braceexpand==0.1.7
 factory-boy==3.3.0
 freezegun==1.4.0
 markdown-it-py>=2.2.0
-pipdeptree==2.16.1
+pipdeptree==2.17.0
 py==1.11.0
 Pygments>=2.15.0  # to bring it up to a secure version
 PyPOM==2.2.4
 pyquery==2.0.0
-pytest==7.4.4  # Included because we use it directly, but also a subdep of pytest-selenium's subdeps
-pytest-cov==4.1.0
+pytest==8.1.1  # Included because we use it directly, but also a subdep of pytest-selenium's subdeps
```

Comment:
I thought this one caused problems?

Reply:
I thought 8.0.0 was the one, but happy to roll back to 7.4.4

Reply:
I trust your work and testing, just jumped out at me b/c I thought I tried some 8.x version but can't remember which. If this works then great!

Reply:
well 7.4.4 works fine right now, too. Can bump it back up next time.